dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ZenodoHelper.java

```diff
@@ -87,33 +261,25 @@ public static ZenodoDoiResult registerZenodoDOI(ApiClient zenodoClient, Workflow
                 returnDeposit = depositApi.createDeposit(deposit);
                 depositionID = returnDeposit.getId();
                 depositMetadata = returnDeposit.getMetadata();
-                // Set the attribute that will reserve a DOI before publishing
```

Comment:
Removed this because the DOI is automatically reserved the first time a deposit is created https://github.com/eblondel/zen4R/issues/37#issuecomment-679846799